recipe/meta.yaml

```diff
@@ -8,10 +8,10 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sshfs-{{ version }}.tar.gz
-  sha256: 14e32f13228bc88c8ccfca03c9a8e508736fa32c303fcf22993c03234a833024
+  sha256: b2b4e0bc50eb00a9ffc3e7fd502bde1aa5fb8321cd8e85deac12e01050f7cae6
 
 build:
-  number: 1
+  number: 0
```

Comment:
@efiop I recall you changed this to 1 in the prev PR, should we revert the change here?

Reply:
@isidentical Yes, it needs to be reset to 0, because the version has changed.